core/src/main/java/org/bitcoinj/wallet/DefaultKeyChainFactory.java

```diff
@@ -32,17 +36,26 @@ public DeterministicKeyChain makeKeyChain(Protos.Key key, Protos.Key firstSubKey
         return chain;
     }
 
+    @Override
+    public DeterministicKeyChain makeKeyChain(Protos.Key key, Protos.Key firstSubKey, DeterministicSeed seed,
+                                              KeyCrypter crypter, boolean isMarried, ImmutableList<ChildNumber> originalAccountPath) {
+        DeterministicKeyChain chain;
+        if (isMarried)
+            chain = new MarriedKeyChain(seed, crypter);
+        else {
```

Comment:
These braces are unnecessary (and inconsistent with the if branch).

Reply:
Corrected.